wiki/Tournaments/Official_support/en.md

```diff
@@ -91,16 +91,37 @@ Once the tournament has concluded, the tournament organisers will need to submit
 - An appropriately designed badge ([see below](#profile-badges)).
 - A link to the publicly viewable catalogue of your tournament's match history.
 
-#### Referee bots/automation
+### Registrant Filtering, Seeding and Automated Refereeing
 
```

Comment:
this looks weird a as a subsection. it's all under the `## Expectations` section anyway

```suggestion
```